paragon/_overrides.scss

```diff
@@ -277,9 +280,16 @@ a.inline-link {
 }
 
 // Progress Bar
-.progress { 
+.progress {
   border: $progress-bar-border-width solid $progress-bar-border-color;
 }
-.progress-bar { 
+.progress-bar {
   background-color: $progress-bar-bg;
 }
+
+// Bubble
+.pgn__bubble {
+  &.pgn__bubble-warning {
+    color: $gray-900;
```

Comment:
Good catch that the text color on the `warning` variant is also not passing valid a11y contrast ratio. Though, it's also not a valid contrast ratio in the core Paragon theme (Open edX) either. If we want it to have a valid contrast ratio for both themes, would it make sense to add this text color change to the core theme?